src/mpid/ch4/src/ch4_init.c

```diff
@@ -300,6 +300,20 @@ static int set_runtime_configurations(void)
 #error "Thread Granularity:  Invalid"
 #endif
 
+static void register_comm_hints(void);
+
+/* Register CH4-specific hints */
+static void register_comm_hints(void)
+{
+    /* Non-standard hints for VCI selection. Default value MPIDI_VCI_INVALID (-1) */
```

Comment:
Why not replace the `-1` below with `MPIDI_VCI_INVALID` (so we don't need comment for it)?